### ISSUE TITLE
[New Format]: Femtonics raw 2p data

### ISSUE THREAD

**Issue by bendichter** [catalystneuro/neuroconv#952]
### What format would you like to see added to NeuroConv?

Femtonics outputs raw 2p data as a .mesc file that is an HDF5 file

### Does the format have any documentation?

Not that I have found but it's pretty straightforward HDF5

### Existing APIs for format

It's HDF5

### Do you have any example files you are willing to share?

I found a .mesc file in https://search.kg.ebrains.eu/instances/a25f722e-f9b1-4f32-80bc-37206c2d1536

Get data > data > calcium > 59911 > 2017-09-29 > MUnit_1 > p29.mesc

### Do you have any interest in helping implement the feature?

Yes.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/catalystneuro/neuroconv/blob/main/.github/CODE_OF_CONDUCT.rst)
- [X] Have you ensured this bug was not already [reported](https://github.com/catalystneuro/neuroconv/issues)?

**Comment by h-mayorquin** [comment]
@bendichter maybe this is a project for @daphnedequatrebarbes?

I can stub the file if you think this is a useful thing to do.

**Comment by bendichter** [comment]
great idea!

**Comment by h-mayorquin** [comment]
Ok, let me stub this.

**Comment by h-mayorquin** [comment]
Here are the stubs, @bendichter 

https://gin.g-node.org/CatalystNeuro/ophys_testing_data/pulls/29

**Comment by bendichter** [comment]
merged

**Comment by h-mayorquin** [comment]
Great. I just added them to the AWS bucket as well. Let me know if you need any help getting hold of the data, @daphnedequatrebarbes.

**Comment by daphnedequatrebarbes** [comment]
In this case do you expect an updated HDF5 extractor that can handle both file types or a new Roiextractor ?

**Comment by daphnedequatrebarbes** [comment]
Yes, that’s what I had in mind — a new extractor that subclasses the existing HDF5 extractor. 
Thx !
